eZ/Publish/Core/Search/Common/FieldRegistry.php

```diff
@@ -20,42 +21,29 @@ class FieldRegistry
      *
      * @var array(string => Indexable)
```

Comment:
@var \eZ\Publish\SPI\FieldType\Indexable[]  ?

Reply:
It is a small very readable class,  I think we do not need extra descriptions/explanations. I would remove also the description of the constructor.